README.md

```diff
@@ -99,3 +99,14 @@ E.g. `#. The placeholder in this string represents the name of a parameter.`
 4. When a PO file reaches 100% translated and reviewed, a webhook pushes it back to the source repo ready to be consumed by your app. 
 
 5. Your app checks the user's locale settings (the browser settings for web apps, or the system settings for the CLI). If we support the user's preferred locale, the app will display strings in the user's language. Otherwise, it defaults to English.
+
+## Merge Pot files rake task
+
+The rake task that merges .pot files is present for the internationalisation of a module. This task uses 'msgcat', which is only natively present on OSes that are GNU based. For running this task locally on another OS you will need to download the gettext pkg and install it locally:
+https://pkgs.org/download/gettext
+
+This task will run within the gettext setup locales_path provided by GettextSetup. The result will be a merged pot file created from all pot files kept in this location.
```

Comment:
I think this would be better if it merged all POT files inside a subdirectory in the locales folder called just `<project_name>`. If this feature is intended to be used only on a per-project basis (multiple POTs in the same project), doing this would make a lot of sense to me!

Reply:
oh i totally read this comment wrong initially. i thought you were proposing we merge all the .pot files inside the locales directory, not just the hard-coded three (ruby, puppet, metadata). it's my understanding that gettext can only use 1 .pot per project so I'm not sure I see the reason for having folders by project_name.

Reply:
To clarify my understanding, are the three POT failes (ruby, puppet, metadata) three separate projects that all output to the `locales` folder currently? Does anything else output to the `locales` folder, such that merging all POT files in that folder could be problematic?

Reply:
My goal here is that this task will merge what we want, without accidentally merging something that happens to use the same folder. I figured since the output is `project_name.pot`, it might make sense to have that file be the result of merging several files in a folder called `project_name`.